content/core-features/authentication.textile

```diff
@@ -70,9 +74,9 @@ Token authentication is typically done in one of four ways:
 
 h3(#token-request-process). Ably TokenRequest is created by your servers and passed to clients
 
-p(tip#timetip). Token requests include a timestamp. So you should ensure that the machine on which you are running your auth server has an accurate clock, e.g. by using an "NTP daemon":https://en.wikipedia.org/wiki/Ntpd . If you are not able to control your machine's clock, you may wish to use the "@queryTime@ auth option":/rest/types#auth-options to avoid "'Timestamp not current' errors":https://knowledge.ably.com/40104-timestamp-not-current .
+p(tip#timetip). Token requests include a timestamp. So you should ensure that the machine on which you are running your auth server has an accurate clock, for example, by using an "NTP daemon":https://en.wikipedia.org/wiki/Ntpd . If you are not able to control your machine's clock, you may wish to use the "@queryTime@ auth option":/rest/types#auth-options to avoid "'Timestamp not current' errors":https://knowledge.ably.com/40104-timestamp-not-current.
```

Comment:
```suggestion
p(tip#timetip). Token requests include a timestamp. So you should ensure that the machine on which you are running your auth server has an accurate clock, for example, by using an "NTP daemon":https://en.wikipedia.org/wiki/Ntpd. If you are not able to control your machine's clock, you may wish to use the "@queryTime@ auth option":/rest/types#auth-options to avoid "'Timestamp not current' errors":https://knowledge.ably.com/40104-timestamp-not-current.
```